src/jdk.jlink/share/classes/jdk/tools/jmod/JmodTask.java

```diff
@@ -1471,7 +1471,7 @@ private void handleOptions(String[] args) {
                 throw new CommandException("err.missing.mode").showUsage(true);
             String verb = words.get(0);
             try {
-                options.mode = Enum.valueOf(Mode.class, verb.toUpperCase());
+                options.mode = Enum.valueOf(Mode.class, verb.toUpperCase(Locale.ROOT));
```

Comment:
The equivalent in jlink and jimage are using Locale.ENGLISH, I guess we should try to keep them consistent.

Reply:
> The equivalent in jlink and jimage are using Locale.ENGLISH, I guess we should try to keep them consistent.

This class is using `Locale.ROOT` to convert modifiers to lowercase, and there is no reference to `Locale.ENGLISH`.

Reply:
My comment was about these cases:
```
src/jdk.jlink/share/classes/jdk/tools/jlink/internal/JlinkTask.java:                        String name = entry.name().toUpperCase(Locale.ENGLISH);
src/jdk.jlink/share/classes/jdk/tools/jimage/JImageTask.java:                options.task = Enum.valueOf(Task.class, args[0].toUpperCase(Locale.ENGLISH));
```